src/Forms/GridField/GridFieldGroupDeleteAction.php

```diff
@@ -41,6 +41,22 @@ public function getColumnContent($gridField, $record, $columnName)
         return null;
     }
 
+    /**
+     * Get the ActionMenu group (not related to Member group)
+     * @param GridField $gridField
+     * @param DataObject $record
+     * @param $columnName
+     * @return null|string
+     */
+    public function getGroup($gridField, $record, $columnName)
+    {
+        if (!$this->canUnlink($record)) {
```

Comment:
Why are we only fixing this bug for groups and not GridFieldDeleteAction? You still need to check `canDelete` / 'canEdit' on that action also. Check the logic in GridFieldDeleteAction::getColumnContent() for the necessary legacy behaviour to replicate.

Reply:
Have migrated this fix to `GridFieldDeleteAction` as well.

Reply:
Thanks good job.